transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipelineBuilder.scala

```diff
@@ -22,6 +22,9 @@ object HcaPipelineBuilder extends PipelineBuilder[Args] {
     metadataEntities.foreach { entityType =>
       processMetadata(ctx, args.inputPrefix, args.outputPrefix, entityType)
     }
+    fileMetadataEntities.foreach { entityType =>
+      processMetadata(ctx, args.inputPrefix, args.outputPrefix, entityType, isFileMetadata = true)
+    }
```

Comment:
The way this is set up, we could have a single list of entities of type `List[(String, Boolean}]` and then only run a single `foreach` over it.

Reply:
```scala
metadataEntities.map(_ -> false) ++ fileMetadataEntities.map(_ -> true)
```
Would save us from typing out a bunch of hard-coded bools